src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/EventPullerServiceTests.cs

```diff
@@ -32,7 +35,7 @@ private static IServiceScopeFactory GivenScopeFactory(ICloudEventHandler handler
 
     private static EventPropagationSubscriptionOptions GivenEventPropagationSubscriptionOptions(IOptionsMonitor<EventPropagationSubscriptionOptions> optionMonitor, string clientName)
     {
-        var option = new AutoFaker<EventPropagationSubscriptionOptions>().RuleFor(x => x.MaxDegreeOfParallelism, f => f.Random.Int(min: 0)).Generate();
+        var option = new AutoFaker<EventPropagationSubscriptionOptions>().RuleFor(x => x.MaxDegreeOfParallelism, f => f.Random.Int(min: 1, max: 10)).Generate();
```

Comment:
Make sure that there is at least one thread. 0 would be problematic because of the current implementation of the PullerService.